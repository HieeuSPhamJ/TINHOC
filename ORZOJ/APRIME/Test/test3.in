288
23670 8580 32268 28363 11582 5315 14024 17259 4122 12856 21390 14315 21358 519 8709 28957 8012 2118 24823 4471 12267 26022 28259 32463 5204 27478 7117 13667 17759 9764 7385 19484 10462 9446 17432 3604 21784 32280 546 19797 10361 22726 23269 21220 2439 31917 17724 4094 25368 11677 10436 29857 12482 22049 28139 3918 12180 17762 21577 23054 1795 1097 820 22707 32295 1508 2190 27744 4392 14091 14818 75 31475 12611 11183 29634 1841 5346 13060 25964 16505 20084 9549 17362 20164 19294 19053 18205 6690 25025 13581 14369 1452 28577 30003 10395 22202 30023 22806 183 7254 12003 12337 1494 10992 29046 17558 16013 83 20207 32389 22513 9175 17812 26173 28690 22232 32420 30738 27077 10600 24635 32448 11687 18241 30325 26623 31838 4349 30678 18263 13858 21898 26858 31542 15229 16344 9000 10304 19278 19668 17180 32486 532 23962 16838 32570 24169 14822 15677 12033 28365 5836 13463 30218 3237 30122 15117 5311 18563 28506 26203 26874 27560 16300 31846 29978 13000 9491 9249 23893 9741 26782 7601 20304 12894 8746 17872 2028 26592 3287 26735 27491 445 32696 9887 25493 6949 24271 29617 30876 13715 6200 28060 22438 29778 420 5280 15364 12607 10761 9769 28286 20176 29592 7926 20202 27477 2544 10362 7165 22910 2193 30083 28000 542 26003 21017 5126 4910 29848 27305 21694 8213 5631 3683 12548 5927 26202 28896 12829 12395 546 12040 27235 14753 12714 6963 8413 25475 4434 5262 23314 29772 2557 28097 14623 887 11717 9934 22536 8905 19693 32316 11852 6289 22294 7542 5820 3887 23734 25710 32140 24019 17607 6277 25515 20103 2748 31407 11344 1392 6329 2258 26838 27984 4611 28272 5833 19792 8630 26758 1089 27645 8630 8372 20078 25488 