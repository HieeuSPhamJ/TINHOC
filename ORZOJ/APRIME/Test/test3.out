0
3
3
81
10
31
7
53
1
14
0
39
17
7
24
80
9
1
58
23
38
0
69
50
4
5
36
27
43
10
36
7
8
11
4
8
13
0
1
40
28
18
63
11
10
49
3
6
1
41
16
87
8
53
83
1
3
9
56
6
15
11
6
46
52
5
0
1
0
30
11
1
58
36
47
1
12
1
16
12
53
10
28
8
12
9
43
83
3
76
18
32
0
58
32
32
5
62
0
2
2
33
46
0
1
0
12
56
4
50
100
65
48
15
79
17
12
13
3
51
7
64
0
35
69
77
81
17
21
3
45
12
19
23
2
67
1
2
14
1
1
11
19
2
15
12
21
84
3
52
28
37
8
52
18
13
13
51
29
64
0
68
1
31
14
8
16
15
34
45
77
23
9
30
2
1
9
7
2
2
19
71
42
8
23
20
69
31
78
69
2
50
8
26
14
1
2
2
13
49
35
55
21
14
0
1
1
52
0
3
46
18
11
84
22
3
58
39
4
7
21
60
12
38
20
15
21
20
2
2
59
53
1
15
86
39
3
22
54
68
0
2
10
1
18
77
62
11
40
8
1
39
69
3
9
28
9
0
0
26
10
3
13
78
48
27
45
37
2
64
12
1
20
11
1
3
21
0
30
7
13
12
8
57
13
6
18
2
